google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/RestoreTableRequest.java

```diff
@@ -29,37 +29,52 @@ public final class RestoreTableRequest {
   private final String sourceBackupId;
   private final String sourceClusterId;
   private final String sourceInstanceId;
+  private final String sourceProjectId;
 
   /**
    * Create a {@link RestoreTableRequest} object. It assumes the source backup locates in the same
-   * instance as the destination table. To restore a table from a backup in another instance, use
+   * instance and project as the destination table. To restore a table from a backup in another instance, use
    * {@link #of(String, String, String) of} method.
    */
   public static RestoreTableRequest of(String sourceClusterId, String sourceBackupId) {
-    RestoreTableRequest request = new RestoreTableRequest(null, sourceClusterId, sourceBackupId);
+    RestoreTableRequest request = new RestoreTableRequest(null, sourceClusterId, sourceBackupId, null);
     return request;
   }
 
   /**
-   * Create a {@link RestoreTableRequest} object. The source backup could locate in a the same or a
-   * different instance.
+   * Create a {@link RestoreTableRequest} object. The source backup could locate in the same or a
+   * different instance but the same project as the destination table. To restore a table from a backup in another project, use
```

Comment:
I think you'll need to reformat the comment here by running: `mvn com.coveo:fmt-maven-plugin:format`